src/utils/fetch-graphql.js

```diff
@@ -443,9 +470,11 @@ const fetchGraphql = async ({
       throw e
     }
 
+    const structuredReporter = makeStructuredReporter(reporter)
```

Comment:
structuredReporter is being created conditionally, but lower down on L504, it's called but is undefined.

I wonder if it would be good to make a reporter helper that gets the reporter, wraps it in structuredReporter, and adds formatting to the input error message 🤔 

Maybe that's outside the scope of this PR, but since all errors need to be wrapped in this PR it might be the best time to do it. It'll make your logic easier too since you could just do:

```
import { wpReporter } from "~/utils/wp-reporter"

wpReporter.error(`Yo this is broken!`)
```

Reply:
@sslotsky , I've added a helper and set the reporter to it using the public Gatsby api instead of directly importing from gatsby-cli